pom.xml

```diff
@@ -83,7 +83,7 @@
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<log4j.version>2.13.3</log4j.version>
 		<vertx.version>4.1.2</vertx.version>
-		<kafka.version>2.7.0</kafka.version>
+		<kafka.version>2.8.0</kafka.version>
```

Comment:
So, why can't we use 2.8.1 here? I understnd that 3.0.0 has issues with VErt.x. But 2.8.1 should be fine here, or?

Reply:
Ahh, I changed it from 2.8.1 to 2.8.0 to check if the test failure were present in this version also. I will bump the version now

Reply:
2.8.1 would be better as it has CVE fixes.

Reply:
I have bumped it to 2.8.1 now